rpm/include.am

```diff
@@ -29,6 +29,14 @@ clean-rpm:
 
 rpm: rpm-build
 
+docker-rpm:
+	@sed -i 's/%{__make} check/#%{__make} check/g' "rpm/spec.in"
```

Comment:
Basically to avoid setting up a MySQL server in the docker container. The check test is already done [here](https://github.com/andreas-bok-sociomantic/libdrizzle-redux/blob/dc05295f05bdd3c3cd8b572e14e3ce7bd441f4ce/.travis.yml#L65)

And it reduces build time